mobile/src/components/courses/components/category-list/style.ts

```diff
@@ -3,7 +3,8 @@ import { StyleSheet } from 'react-native';
 const styles = StyleSheet.create({
   container: {
     flexGrow: 0,
-    paddingHorizontal: 10,
+    marginLeft: 20,
+    marginRight: 10,
```

Comment:
Why not paddingHorizontal: 20 ?

Reply:
List items have marginRight: 10. 
Didn't find solutions with something like `:last-child`